kt/godot-library/src/main/kotlin/godot/core/Properties.kt

```diff
@@ -30,12 +31,21 @@ open class KtProperty<T : KtObject, P : Any?>(
     protected val variantConverter: VariantConverter,
 ) {
     open fun callGet(instance: T) {
-        TransferContext.writeReturnValue(kProperty.get(instance), variantConverter)
+        try {
+            TransferContext.writeReturnValue(kProperty.get(instance), variantConverter)
+        } catch (t: Throwable) {
+            GD.printErr("Error calling a JVM getter from Godot:", t.stackTraceToString())
```

Comment:
Could we also print which instance it is and which property we tried to get?

Reply:
Yeah good point. I just copied-pasted it from the method calls. I forgot to try catch the properties last time I changed the exception checking.
I'll also add more info for method calls.